.github/workflows/ci.yml

```diff
@@ -52,18 +52,17 @@ jobs:
         run: yarn build
   e2e:
     runs-on: ubuntu-latest
-    container: cypress/included:5.0.0
     strategy:
       matrix:
-        browser: [chrome] # cypress executable says 'Browser: 'firefox' was not found on your system or is not supported by Cypress.' seems to be defect
+        browser: [chrome, firefox]
+    container:
+      image: cypress/included:6.0.1
+      options: --user 1001
     env:
       CYPRESS_BROWSER: ${{ matrix.browser }}
     steps:
-      - name: Checkout 🛎
-        uses: actions/checkout@master
-
-      - name: Install dependencies 📦
-        uses: bahmutov/npm-install@v1
-
-      - name: Run tests 🧪
-        run: yarn test:cypress-ci
+      - uses: actions/checkout@v1
+      - uses: cypress-io/github-action@v2.6.1
+        with:
+          start: yarn start
+          browser: ${{ matrix.browser }}
```

Comment:
This is the suggested git action way to run cypress test on CI by [bamutov](https://github.com/cypress-io/cypress-docker-images/issues/363#issuecomment-729948368)